File: find_duplicate_counts.py
def find_duplicate_counts(countries):
    counts = {}
    for country in countries:
        if country in counts:
            counts[country] += 1
        else:
            counts[country] = 1

    duplicates = {}
    for country, count in counts.items():
        if count > 1:
            duplicates[country] = count

    return duplicates


countries = ["Turkey", "Turkey", "India", "USA", "China", "Japan",
             "India", "Russia", "China", "USA", "Canada"]
duplicates = find_duplicate_counts(countries)
print("Duplicate countries and counts:", duplicates)

File: findDuplicateCounts.go
package main

import "fmt"

func findDuplicateCounts(countries []string) map[string]int {
	counts := make(map[string]int)
	for _, country := range countries {
		counts[country]++
	}

	duplicates := make(map[string]int)
	for country, count := range counts {
		if count > 1 {
			duplicates[country] = count
		}
	}

	return duplicates
}

func main() {
	countries := []string{"India", "USA", "China", "Japan", "India", "Russia", "China", "USA", "Canada"}
	duplicates := findDuplicateCounts(countries)
	fmt.Println("Duplicate countries and counts:", duplicates)
}
